internal/app/storage/map.go

```diff
@@ -0,0 +1,56 @@
+package storage
+
+import (
+	"context"
+
+	"github.com/ilya-burinskiy/urlshort/internal/app/models"
+)
+
+type MapStorage map[string][2]string
```

Comment:
```suggestion
type MapStorage map[string]Link
```
лучше использовать понятные типы, тяжело разобраться что такое 0 и 1